.eslintrc.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
     'react-hooks/rules-of-hooks': 'error', // Checks rules of Hooks
     'no-console': 'warn',
     'no-undef': 'off',
-    semi: 'error',
```

Comment:
All of these problems that allow for proper compilation should not prevent a developer from seeing the results of the changes

Reply:
I would not want to have ```warn``` on this rule, it's too crucial to be omitted ocassionally

Reply:
All of these issues could be marked as warnings and be fixed when you compile and push-out the code to the repository. Throwing this as an error breaks the developers feedback loop and disallow you do concentrate on the root of the problem you’re trying to solve. After a few changes of the code you are forced to fight with the linter.